app/channel.go

```diff
@@ -954,7 +954,7 @@ func buildChannelModerations(channelType string, memberRole *model.Role, guestRo
 	}
 
 	var channelModerations []*model.ChannelModeration
-	for _, permissionKey := range model.CHANNEL_MODERATED_PERMISSIONS {
+	for _, permissionKey := range model.ChannelModeratedPermissions {
```

Comment:
is pascal case the preferred case for constants?

Reply:
Yes. See https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps.